### PR TITLE
DoneHW_5.3

### PR DESCRIPTION
**Pull request by MishaBeheka**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -9,6 +15,33 @@
...
+            if (buffer.toString().isEmpty()) {
+                return new String[0];
+            }
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+
+        String[] wordsSeacher = buffer.toString().toLowerCase().split(" ");
+        StringBuilder fishWords = new StringBuilder();
+        for (String i : wordsSeacher) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to create a solution without this loop

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -9,6 +15,33 @@
...
+            if (i.startsWith("w")) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: make "w" as constant of a class

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -9,6 +15,33 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder buffer = new StringBuilder();
+        try (BufferedReader bReader =
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]:
```suggestion
        try (BufferedReader bufferedReader =
```

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (MishaBeheka) [reply src/main/java/core/basesyntax/FileWork.java]: I couldn't change my solution without this loop
I'm sorry(

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -8,7 +14,42 @@
...
+                .split(" ");
+
+        StringBuilder fishWords = new StringBuilder();
+        for (String i : wordsSearcher) {
+            if (i.startsWith(OBJECT_SEARCHER)) {
+                fishWords.append(i).append(" ");
+            }
+        }
+
+        if (fishWords.toString().isBlank()) {
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FileWork.java]: try to solve without this check

**Pull request closed without merging**